project/Dependencies.scala

```diff
@@ -24,15 +23,17 @@ object Dependencies {
   }
 
   object Pekko {
-    private val version     = "1.1.3"
-    val actor               = "org.apache.pekko" %% "pekko-actor"             % version
-    val testkit             = "org.apache.pekko" %% "pekko-testkit"           % version
-    val stream              = "org.apache.pekko" %% "pekko-stream"            % version
-    val persistence         = "org.apache.pekko" %% "pekko-persistence"       % version
-    val `persistence-query` = "org.apache.pekko" %% "pekko-persistence-query" % version
-    val cluster             = "org.apache.pekko" %% "pekko-cluster"           % version
-    val `cluster-sharding`  = "org.apache.pekko" %% "pekko-cluster-sharding"  % version
-    val slf4j               = "org.apache.pekko" %% "pekko-slf4j"             % version
+    private val version       = "1.1.3"
+    val actor                 = "org.apache.pekko" %% "pekko-actor"               % version
+    val testkit               = "org.apache.pekko" %% "pekko-testkit"             % version
+    val stream                = "org.apache.pekko" %% "pekko-stream"              % version
+    val persistence           = "org.apache.pekko" %% "pekko-persistence"         % version
+    val `persistence-testkit` = "org.apache.pekko" %% "pekko-persistence-testkit" % version
+    val `persistence-query`   = "org.apache.pekko" %% "pekko-persistence-query"   % version
+    val cluster               = "org.apache.pekko" %% "pekko-cluster"             % version
+    val `cluster-typed`       = "org.apache.pekko" %% "pekko-cluster-typed"       % version
```

Comment:
`ClusterShardingTest` unit-test required it - we have to check if `pekko-cluster-typed` is required at runtime too.